src/App.test.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import App from './App';
+import { Route } from 'react-router-dom';
```

Comment:
As a best practice, you should always have absolute imports before relative. Easy fix, feel free to reach out for more clarity 👍

Reply:
Updated in latest commit.